workers/unity/Packages/com.improbable.gdk.core/Authority/AuthorityChanges.cs

```diff
@@ -4,6 +4,10 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     This ECS Component stores a ordered list of authority changes received in a single frame.
```

Comment:
should it be a single frame or the current frame?

Reply:
Don't use the word frame. If giving a lifetime then it should say when it's made and destroyed. Or some guarantee a user can use.

Reply:
Also what the order is.

Reply:
Sure can specify the exact lifetime